modernomad/settings/common.py

```diff
@@ -2,33 +2,30 @@
 import os
 import datetime
 import sys
+from pathlib import Path
+import environ
 
-# Make filepaths relative to settings.
-ROOT = os.path.dirname(os.path.abspath(__file__))
-BASE_DIR = os.path.normpath(ROOT + '/..')
+env = environ.Env()
 
-path = lambda *a: os.path.join(ROOT, *a)
+BASE_DIR = Path.cwd()
+BACKUP_ROOT = BASE_DIR / 'backups'
 
-BACKUP_ROOT = ROOT + '/backups/'
+ADMINS = ((
+    env('ADMIN_NAME', default='Unnamed'),
+    env('ADMIN_EMAIL', default='none@example.com')
+),)
 
-ADMINS = (
-    ('Jessy Kate Schingler', 'jessy@embassynetwork.com'),
-)
+MANAGERS = ADMINS
+ALLOWED_HOSTS = env.list('ALLOWED_HOSTS', default=[])
 
-DEBUG = os.getenv('DEBUG', False)
-STRIPE_PUBLISHABLE_KEY = os.getenv('STRIPE_PUBLISHABLE_KEY', default='')
+DEBUG = False
+TEMPLATE_DEBUG = False
 
-MANAGERS = ADMINS
+STRIPE_PUBLISHABLE_KEY = env('STRIPE_PUBLISHABLE_KEY', default='')
+STRIPE_SECRET_KEY = env('STRIPE_SECRET_KEY', default='')
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',  # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
-        'NAME': 'modernomad.db',                      # Or path to database file if using sqlite3.
-        'USER': '',                      # Not used with sqlite3.
-        'PASSWORD': '',                  # Not used with sqlite3.
-        'HOST': '',                      # Set to empty string for localhost. Not used with sqlite3.
-        'PORT': '',                      # Set to empty string for default. Not used with sqlite3.
-    }
+    'default': env.db('DATABASE_URL', default='postgres://postgres@postgres/postgres'),
```

Comment:
```suggestion
    'default': env.db('DATABASE_URL', default='sqlite://modernomad.db'),
```

Then, the `DATABASES` override in `local.py` can be removed.